plasma_framework/migrations/5_deploy_and_register_payment_exit_game.js

```diff
@@ -66,22 +98,36 @@ module.exports = async (
     );
 
     // handle output guard handler
-    const paymentOutputGuardHandler = await PaymentOutputGuardHandler.new(PAYMENT_OUTPUT_TYPE);
+    await deployer.deploy(PaymentOutputGuardHandler, PAYMENT_OUTPUT_TYPE);
+    const paymentOutputGuardHandler = await PaymentOutputGuardHandler.deployed();
     await outputGuardHandlerRegistry.registerOutputGuardHandler(
         PAYMENT_OUTPUT_TYPE, paymentOutputGuardHandler.address,
     );
     await outputGuardHandlerRegistry.renounceOwnership();
 
     // handle spending condition
-    const paymentToPaymentCondition = await PaymentOutputToPaymentTxCondition.new(
-        plasmaFramework.address, PAYMENT_OUTPUT_TYPE, PAYMENT_TX_TYPE,
+    await deployer.deploy(
+        PaymentOutputToPaymentTxCondition,
+        plasmaFramework.address,
+        PAYMENT_OUTPUT_TYPE,
+        PAYMENT_TX_TYPE,
     );
-    const paymentToPaymentV2Condition = await PaymentOutputToPaymentTxCondition.new(
-        plasmaFramework.address, PAYMENT_OUTPUT_TYPE, PAYMENT_V2_TX_TYPE,
+    const paymentToPaymentCondition = await PaymentOutputToPaymentTxCondition.deployed();
+
+    await deployer.deploy(
+        PaymentOutputToPaymentTxCondition,
+        plasmaFramework.address,
+        PAYMENT_OUTPUT_TYPE,
+        PAYMENT_V2_TX_TYPE,
     );
+    const paymentToPaymentV2Condition = await PaymentOutputToPaymentTxCondition.deployed();
```

Comment:
ah, so you can deploy same contract multiple times just when you try to get it back by `deployed()` you will get the latest one?

Reply:
Yeap you can, you save the instance in memory

Reply:
Also `<>.new()` deploys the contracts silently whereas `deployer.deploy(<>)` does it verbosely for some reason.....